docs/0.1/cli_references.md

```diff
@@ -18,3 +18,413 @@ subcommand. The synopsis for each command shows its parameters and their usage.
 
 This usage information is also available on the command line by using the `-h`
 or `--help` option.
+
```

Comment:
Add a `## grid` header here (for easy navigation between the grid and gridd CLIs. (This also will fix the problem of going from the heading level 1 to a heading 3.)